docs/server-side-website.md

```diff
@@ -343,6 +343,19 @@ Applications are of course free to catch errors and display custom error pages.
 
 All headers are forwarded to your Lambda backend. If you used the `forwardedHeaders` option in the past, you can safely remove it.
 
+## Routing paths to other S3 Origins
+
+If you need to route any paths to alternative S3 Origins, you can declare the S3 origin as a source, like you would with a local directly, just prefixing the S3 bucket name with `s3://`.
```

Comment:
```suggestion
You can route any paths to alternative S3 buckets like you would with a local directory. To do that, prefix the S3 bucket name with `s3://`.
```

Can we also add a sentence on how the S3 bucket should be configured? I'm guessing it needs to be either public, or have a origin policy set up?